Sources/StreamChat/Models/Attachments/AnyAttachmentPayload.swift

```diff
@@ -105,7 +118,7 @@ public extension AnyAttachmentPayload {
             .flatMap { try JSONEncoder.stream.encode($0.asAnyEncodable) }
             .flatMap { try JSONDecoder.stream.decode([String: RawJSON].self, from: $0) }
 
-        let payload: AttachmentPayload
+        let payload: any AttachmentPayload
```

Comment:
I have some doubts about using existential types - might be too soon. Swift 5.6 is still new-ish, so not sure if we should start introducing them in the codebase.